src/icons/general/IconCaretDown.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+import { Icon } from '../iconTypes';
+
+const IconCaretDown = ({ className = '', color = '#000', height = 6, title, width = 10 }: Icon) => (
```

Comment:
I suppose we should be importing variables, and using `bdlBlack` here for the default color value, but we're moving away from color stuff, and this icon may already be replaced by one in the new set.

Reply:
@booc0mtaco Yup, this one looks really similar to `Caret8`.  Would you recommend just importing `Caret8` into this icon, for backwards compatibility purposes?

Also, what's the timeline for deprecating these icons?

Reply:
I would avoid that, so the old icons don't change under people.

No date on the deprecation yet. I wouldn't necessarily make any of the color variable changes here, which should keep it simple. If it's valuable, we can do that at a later step!